sql/updates/mangos/z2788_01_mangos_command.sql

```diff
@@ -0,0 +1,8 @@
+DELETE FROM `command` WHERE `name`="list areatriggers";
+INSERT INTO `command` (`name`, `security`, `help`) VALUES
+("list areatriggers", 3, "Syntax: .list areatriggers\n\nShow areatriggers within the same map (if inside an instanceable map) or area (if inside a continent) as the user.");
+
+DELETE FROM `mangos_string` WHERE `entry`=555;
+INSERT INTO `mangos_string` (`entry`, `content_default`) VALUES
+(555, "Showing all areatriggers in %s %s:");
```

Comment:
Double quotes man in all of your commits.

Reply:
I thought double quotes in updates and single quotes in acid db? Sorry, it's hard to break a years old habit :P

Reply:
Everything i know from sql is to never use double quotes ever. Sql standard tells to use single quotes and backticks.

Reply:
https://stackoverflow.com/a/13089425

Reply:
But to be fair we use backticks for object names

Reply:
Got it, thanks! Didn't know about the standard. I'll adapt.